src/utils/common.ts

```diff
@@ -83,6 +83,67 @@ export function getOverrideIP() {
     .trim();
 }
 
+/**
+ * Here we check that the value in the custom param can be considered to be True:
+ * 1. Boolean true value
+ * 2. Any number !== 0
+ * 3. 'yes' or 'true' string in lower / upper case
+ */
+export const hasTrueValue = (value: undefined | null | string | number | boolean): boolean => {
```

Comment:
I would suggest a more verbose name like `isTruthyCustomParamValue` or something like that.